content.html

```diff
@@ -0,0 +1,32 @@
+---
+title: Styling Content
+layout: default
+permalink: /content/
+---
+
+<p>The <code>s-content</code> class can be used on any element where you want to style all the contents of that element. It will apply styles to the paragraphs using <code>.s-p</code> and links using <code>.s-link</code>, so you don't have to add those classes directly to every corresponding element.</p>
+
+<table class="spaced-table">
+  <thead>
+    <tr class="table-head-row">
+      <th>Class Name</th>
+      <th>HTML Example</th>
+      <th>Example Output</th>
+    </tr>
+  </thead>
+  {% for content in site.content %}
+  <tr class="table-row">
+    <td class="util-class-cell">
+      <code>{{ content.value }}</code>
+    </td>
+    <td class="util-value-cell">
+      {% highlight html %}{{ content }}{% endhighlight %}
+    </td>
+    <td class="util-example-cell">
+      <div class="example-container">
+        {{ content | remove: '<p>' | remove: '</p>' }}
+      </div>
+    </td>
+  </tr>
+  {% endfor %}
+</table>
```

Comment:
I think a table here might be overkill, since we're just referencing a single class. Maybe something more like (not exact, but roughly):
```suggestion
{% assign s_content = site.content | first %}

Using <code>{{ s_content.value }}</code>:

{% highlight html %}{{ content }}{% endhighlight %}

{{ content | remove: '<p>' | remove: '</p>' }}
```